src/versionedRouter.js

```diff
@@ -181,12 +181,12 @@ async function compareWithCdkV2(destType, input, feature, v0Result, v0Time) {
   }
 }
 
-async function handleV0Destination(destHandler, destType, input, feature) {
+async function handleV0Destination(destHandler, destType, inputArr, feature) {
```

Comment:
any reason for making this array?

Reply:
As discussed, we are sending an additional argument to the `processRouterDest` method. Please see the PR description for more details.